Composer/scripts/l10n-makeTSV.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const fs = require('fs');
+const path = require('path');
+
+const inPath = process.argv[2];
+
+// eslint-disable-next-line security/detect-non-literal-fs-filename
+const inJson = JSON.parse(fs.readFileSync(inPath));
+const outPath = path.join(path.dirname(inPath), 'en-US.tsv');
+
+// eslint-disable-next-line security/detect-non-literal-fs-filename
+const outTSV = fs.openSync(outPath, 'w');
+fs.writeSync(outTSV, 'Key\tMessage\n');
```

Comment:
We should consider using (fast-csv)[https://c2fo.io/fast-csv] instead to handle escaping properly. We could then use the csv format instead.

Reply:
I just saw the tsv was requested. I still suggest using the `fast-csv` package with the tab delimeter.

Reply:
What they requested was an Excel file. I recommended TSV because none of our strings have tabs in them, making it a little more natural than CSV. You're right, though, that it's probably cleaner to just import a library.

Reply:
Since it is not a requirement to use a tab delimiter, just stick with a comma and allow fast-csv to properly escape the values.